src/components/MentionSuggestions.tsx

```diff
@@ -12,17 +12,17 @@ import Avatar from './Avatar';
 import Text from './Text';
 
 type Mention = {
-    /** Display name of the user */
+    /** Display name of the mention */
     text: string;
 
-    /** The formatted email/phone number of the user */
+    /** The formatted text of the mention */
     alternateText: string;
 
-    /** Email/phone number of the user */
-    login?: string;
+    /** handle of the mention */
```

Comment:
I'm not totally understanding from these prop descriptions what the difference between text, alternateText, and handle is. Maybe we could make these comments a bit more descriptive?